kernel/typing.ml

```diff
@@ -135,11 +135,19 @@ let rec pseudo_u sg (sigma:SS.t) : (int*term*term) list -> SS.t option = functio
       else
         match t1', t2' with
         | Kind, Kind | Type _, Type _ -> pseudo_u sg sigma lst
-        | DB (_,_,n), DB (_,_,n') when ( n=n' ) -> pseudo_u sg sigma lst
-        | Const (_,cst), Const (_,cst') when
-            ( name_eq cst cst' ) ->
-          pseudo_u sg sigma lst
-
+        | DB    (_,_,n), DB    (_,_,n') when n = n'           -> pseudo_u sg sigma lst
+        | Const (_,cst), Const (_,cst') when name_eq cst cst' -> pseudo_u sg sigma lst
+        | Const (l,cst), Const (l',cst') ->
+          if name_eq cst cst'
```

Comment:
Isn't this condition already checked one line upper ?

Reply:
My bad, I forgot to delete the line above. Fixed in the next commit.